pkg/integrations/cmdapi/cmdapi_test.go

```diff
@@ -0,0 +1,25 @@
+package cmdapi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsAllowedToRunStopFromCmdAPI(t *testing.T) {
+	tests := []struct {
+		name            string
+		integrationName string
+		want            bool
+	}{
+		{"empty", "", false},
+		{"allowed", "nri-lsi-java", true},
+		{"not allowed", "", false},
+		{"extra suffix", "nri-lsi-java-foo", false},
```

Comment:
'empty' and 'not allowed' are the same case. And 'extra suffix' is not allowed as well right?